cmd/cape/cmd/key.http

```diff
@@ -0,0 +1 @@
+http://app.capeprivacy.com/v1/user/bendecoste/key
```

Comment:
Is this file being used?

Reply:
oops .. i used it to generate my key.json